wremnants/plot_tools.py

```diff
@@ -7,37 +7,36 @@
 import numpy as np
 hep.style.use(hep.style.ROOT)
 
-def figureWithRatio(href, xlabel, ylabel, ylim, rlabel, rrange, xlim=None):
```

Comment:
Is this an intentional change?

Reply:
This is accidental, reverting the change